### DISCUSSION TITLE
Charcoal pit

### DISCUSSION THREAD

**Discussion by LucasAG95** [sunflower-land/sunflower-land#3424] [Ideas]
*"I don't speak English, and I used a translator; there may be errors in the text. I apologize in advance."

I see that many players are asking for wood burning and ways to speed up our kitchens, so with that in mind, I had this idea of a Charcoal Pit.
The idea itself is similar to compote makers where we obtain bait. The idea is to create a building or a machine that manufactures charcoal, where we have to burn wood to produce it. We can consider the idea of burning other resources, but I believe wood would be essential.
The charcoal would be used in food buildings to accelerate the cooking time. I thought of a reduction of 15% to 25% (the idea of 25% appeals to me more, but I'm not sure if it's feasible or not). Each building or food item would require a certain amount of charcoal.

Example 1:

- Fire Pit - 10 Charcoals
- Kitchen - 20 Charcoals
- Bakery - 40 Charcoals
- Deli - 25 Charcoals

Example 2:

- Foods that take up to 4 hours - 10 Charcoals
- Foods that take up to 8 hours - 15 Charcoals
- Foods that take up to 12 hours - 20 Charcoals
- Foods that take up to 24 hours - 40 Charcoals

*I didn't add the Smoothie Shack because it doesn't make sense to use charcoal in it lol

I didn't think of a specific amount of wood to burn, but maybe 5 wood per charcoal? I think that would be a reasonable number, and we can also increase it according to how many charcoals the person is making at once in the "machine," for example:

- 10 Charcoals = 50 Wood - 8 hours to be ready
- 20 Charcoals = 110 Wood - 12 hours to be ready
- 30 Charcoals = 170 Wood - 16 hours to be ready
- 40 Charcoals = 240 Wood - 20 hours to be ready
- 50 Charcoals = 310 Wood - 24 hours to be ready
- 100 Charcoals = 700 Wood - 36 hours to be ready

In this example, the person would burn more wood each time they wanted more charcoals at once, but at the same time, it would reduce the waiting time by the same amount. Someone who chooses to make 10 Charcoals up to 100 would save 200 wood but take an additional 72 hours.

*Time and wood spent can be adjusted; this is just an example, although I like the values.

I believe that the quantity and time for charcoal production wouldn't make it "overpowered" as players would take time to obtain the charcoal to use in a building, besides the fact that each building requests a certain amount.

Ideas for SFTs:

- "Pet" little charcoal: You can have 2 charcoal pits.
- Maybe an apron that reduces the charcoal collection time by 50% (yes, it seems overpowered, but it would also use double the wood).
- Charcoal scoop: 10% more.

I believe there is a lot to explore, I didn't include SFTs to reduce wood because the goal is to burn them (literally lol), so it's better to obtain more charcoal than to burn less wood.

**Comment by raulmonoanivia** [comment]
100% agreed please turn this into reality asap

**Comment by TheDcol** [comment]
Nice post mate! 
It seems like it's an added version to this: <https://github.com/sunflower-land/sunflower-land/discussions/3404>

1 question sorry if I missed it, would adding charcoal to a building where I am cooking a food instantly decrease the time or would the player need to add charcoal first, then the food? the 15-25% would be flat across all the food buildings?

**Reply by LucasAG95** [reply]
The 15% or 25% would be fixed, like Nancy for crops
The coals could be added later I believe, like eggs in the compost bins, although it makes more sense to put it first

**Reply by LucasAG95** [reply]
The worst thing is that I saw your proposal, but I didn't remember, this coal thing stuck in my head, maybe your proposal was a motivation for me to think about it :D

**Reply by TheDcol** [reply]
All good, i like the charcoal addition, adds a bit more depth if it's something the devs want to pursue (would require more coding/art I assume rather than just have straight wood, but i like it)

**Comment by rasheed19871** [comment]
That's great idea 😄 I am sure it will go to the game it's so perfect they wouldn't resist, love it😜 this project need intelligent mind and luckily we got a few Adam plz take a look
